composer.json

```diff
@@ -7,5 +7,9 @@
     "require": {
         "civicrm/civicrm-drupal-8": "*"
     },
+    "require-dev": {
+        "drupal/fullcalendar_view": "^5.0",
+        "drupal/leaflet": "^2"
+    },
```

Comment:
The GitHub Actions workflow doesn't parse this, but DrupalCI does. I know we do not run with DrupalCI, but adding for "just in case".